src/software/simulated_tests/validation_functions/robots_avoid_ball_validation.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/world/world.h"
+
+/**
+ * Checks if robots are keeping at least 0.5 meters from
+ * the ball
+ * @param world_ptr the world pointer given by the simulator. Gets updated every tick
+ * @param yield yields control to the next routine (coroutines)
+ */
+void robotsAvoidBall(std::shared_ptr<World> world_ptr,
```

Comment:
can we add a parameter called tolerance or something so that we're not only limited to 0.5m?

Reply:
Can you clarify what you mean by adding tolerance? The stop play requires specifically 0.5m distance, but I could change it so the validation function takes in a parameter rather than using a constant defined in the validation function so that other plays than the stop plays could use this validation function for different distances.

Reply:
@tarakong01 yup that's great, that's what I meant

Reply:
Added!